qprofiler/test/org/qcmg/qprofiler/QProfilerTest.java

```diff
@@ -39,6 +40,17 @@ public void setup() throws IOException {
 		
 		createTestFile(DODGY_GFF_FILE_NAME_FILE, getDodgyFileContents());		
 	}
+	
+	@After
+	public void tidyUp() throws IOException {
```

Comment:
It's generally safer to use the `TemporaryFolder` class to setup files in each test method, as that way, junit takes care of removing the files once they have been used. I know that in this case the original code is not yours - do you know which test method is producing the `qprofiler.xml` file? If so, perhaps that method(s) can be updated to take a temp file as the output?

Reply:
qprofiler.xml is a default output if the --output option is not called, see details on QProfiler::setup() {...if (null != options.getOutputFileName()) ...}. 
I think it is a good idea to create a default output, but it should not be always tested. The unit test is updated, we test qprofiler with --output option, unless once to test the default one exits or not.